### PR TITLE
Avatar Volume Controls

### PR DESCRIPTION
**Pull request by gfodor**
You can now adjust the voice volume of avatars and also focus/track them with your camera.

image

### CODE REVIEW

File: src/components/avatar-volume-controls.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  updateVolumeLabel() {
+    this.volumeLabel.setAttribute(
+      "text",
+      "value",
+      this.data.volume === 0 ? "Muted" : VOLUME_LABELS[Math.floor(this.data.volume / 0.05)]
+    );
+  },
+
+  tick() {
```
Comment (netpro2k) [inline src/components/avatar-volume-controls.js]: Might be able to replace this tick and `parentNode.parentNode` with a `  NAF.utils.getNetworkedEntity(this.el)` in init.

File: src/hub.html [hunk 2]
```diff
@@ -156,12 +156,37 @@
...
-                                <a-entity billboard matrix-auto-update visibility-while-frozen="withinDistance: 10; requireHoverOnNonMobile: false;">
-                                    <a-entity is-remote-hover-target tags="singleActionButton:true;" mixin="rounded-text-button" block-button ui-class-while-frozen position="0 0 .35">
-                                        <a-entity text="value:Hide; width:2.5; align:center;" position="0 0 0.01"></a-entity>
-                                    </a-entity>
-                                    <a-entity is-remote-hover-target tags="singleActionButton:true;" mixin="rounded-text-button" visible-if-permitted="kick_users" kick-button ui-class-while-frozen position="0 -0.25 .35">
-                                        <a-entity text="value:Kick; width:2.5; align:center;"  visible-if-permitted="kick_users" position="0 0 0.01"></a-entity>
+                                <a-entity class="ui interactable-ui">
+                                    <a-entity matrix-auto-update billboard class="freeze-menu" visibility-while-frozen="withinDistance: 100; requireHoverOnNonMobile: false;" avatar-volume-controls>
```
Comment (netpro2k) [inline src/hub.html]: Would be nice if we can ditch `matrix-auto-update` here

Comment (johnshaughnessy) [reply src/hub.html]: +1

Comment (gfodor) [reply src/hub.html]: this was here before, i believe due to `billboard` part of `aframe-billboard-component`

**Pull request merged**